src/ucs/arch/ppc64/cpu.h

```diff
@@ -27,7 +27,7 @@ BEGIN_C_DECLS
 #define ucs_memory_bus_fence()        asm volatile ("sync"::: "memory")
 #define ucs_memory_bus_store_fence()  ucs_memory_bus_fence()
 #define ucs_memory_bus_load_fence()   ucs_memory_bus_fence()
-#define ucs_memory_bus_wc_flush()
+#define ucs_memory_bus_cacheline_wc_flush()
```

Comment:
what is the reason to rename?

Reply:
To indicate that this a special flush that used for pushing out the entire c$ line. You cannot use this flush with any other combination of stores.